ChronoBank/ChronoMint
# MINT-61 wallet reworked and global refinements

**Work in progress.**

What is done:
- new working wallet transactions scanner
- transfers watcher
- refinements for whole wallet code
- wallet notifications for sender and receiver
- AbstractContractDAO._watch fixed to watch all necessary events and ignore already watched
- no more utils directory
- all forms code (except for NewPollForm) refined
- _window.localStorage.chronoBankAccount_ AND _window.localStorage.getItem('chronoBankAccount'))_ changed to window.localStorage.account
- many other global refinements

Remained:
~~1. Test~~
~~- requireTIME~~
~~- depositTIME~~
~~- withdrawTIME~~
2. Notifications for ETH transfer and transferETH reworking
~~3. Check all fetching flags/loaders~~
4. Use name.call and decimals.call to retrieve token data
5. Specs
6. Check LHT transfer
7. Exchange section
